client/app/app.tpl.less

```diff
@@ -51,6 +51,7 @@
 @brand-info: #3f5167;
 @gray-base: #757575;
 @gray-dark: #333;
+@gray-silver: #959595;
```

Comment:
It sounds really near from `@oui-color-dusty` right?

Reply:
Yes, but those colors have nothing to do with UI-KIT
@gray-silver is following naming for greys inside bootstrap, It's use for breadcrumbs and UI-kit don't have breadcrumbs, but ui-kit-bs have it see https://ovh-ux.github.io/ovh-ui-kit-bs/#breacrumb

Reply:
@gray-base, @gray-dark, @gray-light, @gray-lighter are also heritage from bootstrap